package.json

```diff
@@ -89,9 +89,10 @@
     "hyperhtml": "^2.10.0",
     "loading-indicator": "^2.0.0",
     "marked": "^0.3.19",
+    "npm": "^6.0.1",
```

Comment:
Should we do this, or `"npm": ">=5"` in `engines` field instead?

Reply:
Or just skip this line? I'm not sure installing npm as a dependency is right here...

Reply:
agree.